third_party/optimized-svg-icons/amp-social-share-svgs.css

```diff
@@ -21,3 +21,7 @@
 }
+
+.amp-social-share-whatsapp a {
+    background-image: url('data:image/svg+xml;utf8,%3Csvg%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%20x%3D%220px%22%20y%3D%220px%22%0A%09%20viewBox%3D%220%200%20512%20514.5%22%20style%3D%22enable-background%3Anew%200%200%20512%20514.5%3B%22%20xml%3Aspace%3D%22preserve%22%3E%0A%3Cstyle%20type%3D%22text%2Fcss%22%3E%0A%09.st0%7Bdisplay%3Anone%3B%7D%0A%09.st1%7Bdisplay%3Ainline%3Bfill%3A%2325D366%3B%7D%0A%09.st2%7Bfill-rule%3Aevenodd%3Bclip-rule%3Aevenodd%3Bfill%3A%23FFFFFF%3B%7D%0A%3C%2Fstyle%3E%0A%3Cg%20id%3D%22background_1_%22%20class%3D%22st0%22%3E%0A%09%3Cpath%20id%3D%22background%22%20class%3D%22st1%22%20d%3D%22M876.2%2C788.2H-364.4c-11%2C0-19.9-8.9-19.9-19.9V-258.8c0-11%2C8.9-19.9%2C19.9-19.9H876.2%0A%09%09c11%2C0%2C19.9%2C8.9%2C19.9%2C19.9V768.4C896%2C779.3%2C887.1%2C788.2%2C876.2%2C788.2z%22%2F%3E%0A%3C%2Fg%3E%0A%3Cg%20id%3D%22WhatsApp_Logo%22%3E%0A%09%3Cg%20id%3D%22WA_Logo%22%3E%0A%09%09%3Cg%3E%0A%09%09%09%3Cpath%20class%3D%22st2%22%20d%3D%22M404.4%2C108C365%2C68.5%2C312.7%2C46.8%2C256.9%2C46.8c-115%2C0-208.5%2C93.6-208.6%2C208.5c0%2C36.8%2C9.6%2C72.6%2C27.8%2C104.3%0A%09%09%09%09L46.6%2C467.7l110.6-29c30.5%2C16.6%2C64.8%2C25.4%2C99.7%2C25.4h0.1c0%2C0%2C0%2C0%2C0%2C0c114.9%2C0%2C208.5-93.6%2C208.6-208.6%0A%09%09%09%09C465.5%2C199.8%2C443.8%2C147.4%2C404.4%2C108z%20M256.9%2C428.8L256.9%2C428.8c-31.2%2C0-61.7-8.4-88.3-24.2l-6.3-3.8l-65.6%2C17.2l17.5-64l-4.1-6.6%0A%09%09%09%09c-17.4-27.6-26.5-59.5-26.5-92.2C83.6%2C159.8%2C161.3%2C82%2C256.9%2C82c46.3%2C0%2C89.8%2C18.1%2C122.6%2C50.8c32.7%2C32.8%2C50.7%2C76.3%2C50.7%2C122.6%0A%09%09%09%09C430.2%2C351.1%2C352.4%2C428.8%2C256.9%2C428.8z%20M352%2C299c-5.2-2.6-30.8-15.2-35.6-17c-4.8-1.7-8.3-2.6-11.7%2C2.6%0A%09%09%09%09c-3.5%2C5.2-13.5%2C17-16.5%2C20.4c-3%2C3.5-6.1%2C3.9-11.3%2C1.3c-5.2-2.6-22-8.1-41.9-25.9c-15.5-13.8-25.9-30.9-29-36.1%0A%09%09%09%09c-3-5.2-0.3-8%2C2.3-10.6c2.3-2.3%2C5.2-6.1%2C7.8-9.1c2.6-3%2C3.5-5.2%2C5.2-8.7c1.7-3.5%2C0.9-6.5-0.4-9.1c-1.3-2.6-11.7-28.3-16.1-38.7%0A%09%09%09%09c-4.2-10.2-8.5-8.8-11.7-8.9c-3-0.2-6.5-0.2-10-0.2c-3.5%2C0-9.1%2C1.3-13.9%2C6.5c-4.8%2C5.2-18.2%2C17.8-18.2%2C43.5%0A%09%09%09%09c0%2C25.7%2C18.7%2C50.4%2C21.3%2C53.9c2.6%2C3.5%2C36.7%2C56.1%2C89%2C78.7c12.4%2C5.4%2C22.1%2C8.6%2C29.7%2C11c12.5%2C4%2C23.8%2C3.4%2C32.8%2C2.1%0A%09%09%09%09c10-1.5%2C30.8-12.6%2C35.2-24.8c4.3-12.2%2C4.3-22.6%2C3-24.8C360.6%2C302.9%2C357.2%2C301.6%2C352%2C299z%22%2F%3E%0A%09%09%3C%2Fg%3E%0A%09%3C%2Fg%3E%0A%3C%2Fg%3E%0A%3C%2Fsvg%3E');
```

Comment:
This very file is Apache 2.0 licensed. We can accept other compatible licenses, but without that it cannot go into this repo. It would probably be best if someone from WhatsApp could contribute the resource.

@adewale Do you happen to know who we could ping?